Makefile

```diff
@@ -11,9 +11,15 @@ help:
 	@echo ''
 
 build:
+	python hooks/flat_schedule.py
+	python hooks/guidebook.py
+	python hooks/schedule_summary.py
```

Comment:
They can be removed. I did that while trying to workaround the hooks not running. Fixing it.